### ISSUE TITLE
Ability to only enable specific core plugins

### ISSUE THREAD

**Issue by benface** [tailwindcss/tailwindcss#850]
Right now you can disable all core plugins with `corePlugins: false` (thank you @adamwathan!) but [as Adam said](https://github.com/tailwindcss/tailwindcss/issues/833#issuecomment-484162018), it would be useful to have a "whitelisting" option to only enable one or a few core plugins in particular, without worrying that a Tailwind update might change the code's behaviour by introducing new core plugins.

An idea, perhaps the `corePlugins` option could take accept an object formatted like this:
```
corePlugins: {
    only: [
        'backgroundColor',
        'textColor',
    ],
}
```

**Comment by hacknug** [comment]
Would it make sense to also add a "blacklisting" option?

**Comment by benface** [comment]
@hacknug All core plugins are enabled by default, so to blacklist you only have to do:
```
corePlugins: {
    'backgroundColor': false,
    'textColor': false,
}
```

**Comment by hacknug** [comment]
Silly me. I keep mixing the old and new configs structures in my head haha

**Comment by adamwathan** [comment]
What about just an array?

```js
corePlugins: [
  'backgroundColor',
  'textColor',
]
```

**Comment by benface** [comment]
@adamwathan Oh yeah, that’s much better haha.

**Comment by adamwathan** [comment]
Solved in #853.
